R/Run_Mod.R

```diff
@@ -60,18 +81,22 @@
 #'   - An optional `"jags.post"` attribute, included when argument
 #'   `with_post` = TRUE.
 #' @inheritDotParams prep_priors
+#' @seealso clean_priors, prep_priors_multi_b, inits_kron, write_model_ch2_kron
```

Comment:
please cross-reference wherever possible:
```suggestion
#' @seealso [clean_priors], [prep_priors_multi_b], [inits_kron], [write_model_ch2_kron]
```